components/classroom/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Classroom Page</title>
+</head>
+
+<body>
+    <h1>Classroom Group Management</h1>
+    <p id="project-info">Project ID: </p>
+
+
+    <script src="project.js"></script>
+
+    <script>
```

Comment:
Move this script tag into the 'project.js' file since you already have a js file created. It isn't a big deal but makes the html file cleaner.